src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -107,36 +114,63 @@ void RotatedSPOs::extractParamsFromAntiSymmetricMatrix(const RotationIndices& ro
 
 void RotatedSPOs::resetParametersExclusive(const opt_variables_type& active)
 {
-  std::vector<ValueType> delta_param(m_act_rot_inds_.size());
+  const size_t N = m_act_rot_inds_.size();
+  std::vector<ValueType> delta_param(N);
 
-  size_t psize = m_act_rot_inds_.size();
+  size_t psize = N;
 
   if (use_global_rot_)
   {
     psize = m_full_rot_inds_.size();
-    assert(psize >= m_act_rot_inds_.size());
+    assert(psize >= N);
   }
 
   std::vector<ValueType> old_param(psize);
   std::vector<ValueType> new_param(psize);
 
-  for (int i = 0; i < m_act_rot_inds_.size(); i++)
+#ifdef QMC_COMPLEX
+  for (int i = 0; i < N; i++)
+  {
+    int loc0       = myVars.where(i);
+    int loc1       = myVars.where(i + N);
```

Comment:
I feel alternating real and imaginary parts straightforward why choosing all real + all imaginary?

Reply:
This was just a choice that was made so that the we could leave the real parts of the code as is in some places and then add the complex part after with an offset. We can change it if necessary

Reply:
> we could leave the real parts of the code as is in some places

Could you point me? offset +N adds difficulty writing code with iterators. There is no performance concern.